js-functions_pizza-analyzer/js/index.js

```diff
@@ -7,20 +7,43 @@ const pizza2 = document.querySelector('[data-js="pizza-2"]');
 const outputSection = document.querySelector('[data-js="output-section"]');
 
 const output = document.querySelector('[data-js="output"]');
-
+let pizzaSize1;
+let pizzaSize2;
 pizzaInput1.addEventListener("input", () => {
   // write your code here
+  pizzaSize1 = pizzaInput1.value;
+  updatePizzaDisplay(pizza1, pizzaSize1);
 });
 
 pizzaInput2.addEventListener("input", () => {
   // write your code here
+  pizzaSize2 = pizzaInput2.value;
```

Comment:
The instructions state "Inside both event listeners, save the current values of both pizza inputs in two variables `pizzaSize1` and `pizzaSize2`" 

And this is actually necessary to get the program to work.